fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -162,7 +173,7 @@ public IDatabaseTranslator getTranslator() {
      * @param pdm
      */
     public void createSchema(PhysicalDataModel pdm) {
-        createSchema(new AllVersionHistoryService(), pdm);
+        createSchema(vhs, pdm);
```

Comment:
good to see that we don't create the versionhistoryService every time calling createSchema now!
just double check, createSchema is never called before runWithAdapter(), correct?

Reply:
indirectly - yes